tests/epyccel/modules/tuples.py

```diff
@@ -88,7 +88,9 @@ def inhomogenous_tuple_3():
     return ai[0], ai[1], ai[2]
 
 def inhomogenous_tuple_2_levels_1():
-    ai = ((1,2), (4,False), (3.0, 'boo'))
+    # TODO [EB 15.06.21] Put back original test when strings are supported
+    #ai = ((1,2), (4,False), (3.0, 'boo'))
+    ai = ((1,2), (4,False), (3.0, True))
```

Comment:
How was the original test working?

Reply:
Fortran supports strings. As this is an inhomogeneous tuple it causes no problems in fortran, but strings aren't supported in C at all

Reply:
Maybe we should write "when strings are supported **in C**" then...

Reply:
Fixed 62ec0d7